src/main/java/org/broadinstitute/hellbender/tools/exome/allelefraction/AllelicBiasParameters.java

```diff
@@ -0,0 +1,53 @@
+package org.broadinstitute.hellbender.tools.exome.allelefraction;
+
+/**
+ * Encapsulates the global parameters of the allele fraction model: the mean and variance of the common prior on
```

Comment:
Hmm, I'm a bit conflicted about this refactoring.  On the one hand, it's nice to have these parameters packaged together.  On the other, it's not always clear when one should be using `AlleleFractionState` vs. `AllelicBiasParameters` + `AlleleFractionState.MinorFractions` (for example, in `AlleleFractionLikelihoods.logLikelihood`).  Let me think more about it as I go through the review!